python/dolfinx/jit.py

```diff
@@ -21,7 +21,9 @@
 
 DOLFINX_DEFAULT_JIT_PARAMETERS = {
     "cache_dir":
-        (Path.joinpath(Path.home(), ".cache", "fenics"), "Path for storing DOLFINx JIT cache"),
+        (os.getenv("XDG_CACHE_HOME", default=Path.home().joinpath(".cache")) / Path("fenics"),
+         "Path for storing DOLFINx JIT cache."
+         + "Default prefix ~/.cache/ can be changed using XDG_CACHE_HOME environment variable."),
```

Comment:
`+` is not necessary

```python3
a = (1, "hello" "world")
print(a)
```
```
(1, 'helloworld')
```

Reply:
Thanks, flake8 had me running round in circles.

Reply:
+)